### DISCUSSION TITLE
Bluesky logo not displaying

### DISCUSSION THREAD

**Discussion by nickader** [greenelab/lab-website-template#282] [Q&A] [answered]
### Checks

- [X] I have searched **[the docs](https://greene-lab.gitbook.io/lab-website-template-docs)**, [existing issues](https://github.com/greenelab/lab-website-template/issues), and [existing discussions](https://github.com/greenelab/lab-website-template/discussions) for answers first.

### Link to your website repo

https://github.com/nickader/AderLabWebsite/tree/main

### Version of Lab Website Template you are using

1.2.1

### Description

Trying to get Bluesky to show up as a social link. It's there and clickable, but the icon is not showing. When I change the icon called to another used icon (like fa-user-graduate), then that icon shows up. I saw [this discussion](https://github.com/greenelab/lab-website-template/discussions/279) but that doesn't seem to be related to the issue of the bluesky icon as far as I can tell. Thanks for the help!

**Comment by vincerubinetti** [comment, marked as answer]
You have a typo in your icon definition:

![image](https://github.com/user-attachments/assets/9a59fc98-b0ae-4feb-a0d7-0c8e6a3ac6f5)

**Comment by nickader** [comment]
Ooof. Well, that is embarrassing. Thank you for taking the time to help silly users like me! This template is a huge asset!!!
